<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidget</class>
 <widget class="QWidget" name="ChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>局域网聊天工具</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="11">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>929</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>929</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Chat Room</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>0</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="IconSources.qrc">
        <normaloff>:/new/icon/images/minmin.png</normaloff>:/new/icon/images/minmin.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>0</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="IconSources.qrc">
        <normaloff>:/new/icon/images/close.png</normaloff>:/new/icon/images/close.png</iconset>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="9">
    <widget class="QTextBrowser" name="messageBrowser"/>
   </item>
   <item row="1" column="9" rowspan="3" colspan="2">
    <widget class="QTableWidget" name="userTableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>昵称</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFontComboBox" name="fontComboBox"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="sizeComboBox">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="boldToolBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>加粗</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="IconSources.qrc">
       <normaloff>:/new/icon/images/bold.png</normaloff>:/new/icon/images/bold.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="italicToolBtn">
     <property name="toolTip">
      <string>倾斜</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="IconSources.qrc">
       <normaloff>:/new/icon/images/italic.png</normaloff>:/new/icon/images/italic.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QToolButton" name="underlineToolBtn">
     <property name="toolTip">
      <string>下划线</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="IconSources.qrc">
       <normaloff>:/new/icon/images/under.png</normaloff>:/new/icon/images/under.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QToolButton" name="colorToolBtn">
     <property name="toolTip">
      <string>更改字体颜色</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IconSources.qrc">
       <normaloff>:/new/icon/images/color.png</normaloff>:/new/icon/images/color.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QToolButton" name="sendToolBtn">
     <property name="toolTip">
      <string>传输文件</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IconSources.qrc">
       <normaloff>:/new/icon/images/send.png</normaloff>:/new/icon/images/send.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QToolButton" name="saveToolBtn">
     <property name="toolTip">
      <string>保存聊天记录</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IconSources.qrc">
       <normaloff>:/new/icon/images/save.png</normaloff>:/new/icon/images/save.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QToolButton" name="clearToolBtn">
     <property name="toolTip">
      <string>清空聊天记录</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IconSources.qrc">
       <normaloff>:/new/icon/images/clear.png</normaloff>:/new/icon/images/clear.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="9">
    <widget class="QTextEdit" name="messageTextEdit"/>
   </item>
   <item row="4" column="6" colspan="3">
    <widget class="QLabel" name="userNumberLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="9">
    <widget class="QPushButton" name="sendButton">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="4" column="10">
    <widget class="QPushButton" name="exitButton">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>userTableWidget</tabstop>
  <tabstop>fontComboBox</tabstop>
  <tabstop>sizeComboBox</tabstop>
  <tabstop>messageBrowser</tabstop>
  <tabstop>boldToolBtn</tabstop>
  <tabstop>italicToolBtn</tabstop>
  <tabstop>underlineToolBtn</tabstop>
  <tabstop>colorToolBtn</tabstop>
  <tabstop>sendToolBtn</tabstop>
  <tabstop>saveToolBtn</tabstop>
  <tabstop>clearToolBtn</tabstop>
  <tabstop>messageTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="IconSources.qrc"/>
 </resources>
 <connections/>
</ui>
